src/boot/efi/secure-boot.c

```diff
@@ -109,7 +109,7 @@ EFI_STATUS secure_boot_enroll_at(EFI_FILE *root_dir, const char16_t *path) {
 
                 err = efivar_set_raw(&sb_vars[i].vendor, sb_vars[i].name, sb_vars[i].buffer, sb_vars[i].size, sb_vars_opts);
                 if (err != EFI_SUCCESS) {
-                        log_error_stall(L"Failed to write %s secure boot variable: %r", sb_vars[i].name, err);
+                        log_error("Failed to write %ls secure boot variable: %m", sb_vars[i].name);
```

Comment:
Hmm, `%m` should only be used with `log_*_errno`. `err` seems to have been lost.

Reply:
And here I thought I've stared enough times at these conversion to think I've fixed them all. :D